docs/source/gbd2019_models/risk_effects/bmi/index.rst

```diff
@@ -0,0 +1,312 @@
+
+  Section Level 4
+  ~~~~~~~~~~~~~~~
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+======================================
+Body Mass Index
+======================================
+
+
+.. contents::
+   :local:
+   :depth: 2
+
+Risk Overview
+-------------
+
+Body mass index (BMI) is a person’s weight in kilograms divided by the square of height in meters (weight (kg) / [height (m)]\ :sup:`2`\). BMI is an inexpensive and easy screening method; values are frequently divided into categories: underweight (<18.5 kg/m\ :sup:`2`\), healthy weight (18.5 to 24.9 kg/m\ :sup:`2`\), overweight (25.0 to 29.9 kg/m\ :sup:`2`\), and obese (>=30.0 kg/m\ :sup:`2`\). BMI does not measure body fat directly, but it is moderately correlated with more direct measures of body fat. BMI has also been shown to be correlated with various metabolic and disease outcomes. BMI can be a screening tool, but it does not diagnose the body fatness or health of an individual.
+[CDC-BMI2]_
+
+Obesity, and body fat are known to be associated with heart disease. 
+"The mechanisms through which obesity increases CVD risk involve 
+changes in body composition that can affect hemodynamics and alters 
+heart structure." [Carbone_2019]_ While BMI is not a direct measure of 
+excess fat, it is correlated and therefore a useful proxy and easy to 
+measure across individuals. 
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+Risk exposure
++++++++++++++
+
+:ref:`See risk exposure documentation for SBP <2019_risk_bmi>`
+
+Relative risks
+++++++++++++++
+
+BMI is a risk factor for many causes which are listed in the table 
+below. For the CVD model, BMI will affect ischemic heart disease, 
+ischemic stroke, and heart failure. There were no updates to GBD 
+modeling of BMI for these causes. 
+
+RRs were reported per 5 kg/m\ :sup:`2`\  increase in BMI above the TMREL value 
+(20–25 kg/m\ :sup:`2`\), calculated as in the equation below: 
+
+:math:`\text{RR(x)} = {\text{RR}_0}^{\frac{\max\left((x-\text{TMREL}), 0\right)}{\text{5 kg/m2}}}`
+
+Where RR(x) is the RR at exposure level x and RR\ :sub:`0`\  is the increase in RR for each 5 kg/m\ :sup:`2`\ above the TMREL. We used DisMod-MR 2.1 to pool effect sizes from included studies and generate a dose-response curve for each of the outcomes associated with high SBP. The tool enabled us to incorporate random effects across studies and include data with different age ranges. RRs were used universally for all countries and the meta-regression only helped to pool the three major sources and produce RRs with uncertainty and covariance across ages taking into account the uncertainty of the data points. 
```

Comment:
This is a small point, but I think "GBD" may be slightly more clear than "we" because I always interpret "we" to mean simulation science in these docs!

Reply:
Hahah great catch! I think Catherine wrote this actually and meant "we" to be GBD because that is "we" for her. But yes, I agree this could be more clear as "GBD"